torch/onnx/symbolic_opset10.py

```diff
@@ -145,12 +146,12 @@ def _avg_pool(name, tuple_fn):
     @symbolic_helper.parse_args("v", "is", "is", "is", "i", "i", "none")
     def symbolic_fn(
         g,
-        input,
-        kernel_size,
-        stride,
-        padding,
-        ceil_mode,
-        count_include_pad,
+        input: _C.Value,
+        kernel_size: Tuple[int, ...],
+        stride: Tuple[int, ...],
+        padding: Union[int, Tuple[int, ...]],
```

Comment:
is it possible for padding to have type `int`? It's also parsed by `is`.

Reply:
Changed to Sequence[int]